README.md

```diff
@@ -333,6 +334,22 @@ When `enableLowInitialPlaylist` is set to true, it will be used to select
 the lowest bitrate playlist initially.  This helps to decrease playback start time.
 This setting is `false` by default.
 
+##### smoothQualityChange
+* Type: `boolean`
+* can be used as a source option
+* can be used as an initialization option
+
+When the `smoothQualityChange` property is set to `true`, a manual quality
+change triggered via the `qualityLevels` API will use smooth quality switching
+rather than the default fast (buffer-ejecting) quality switching. Using smooth
+quality switching will mean no loading spinner will appear during quality
+switches, but will cause quality switches to only be visible after a few
+seconds.
+
+Note that this _only_ affects quality changes triggered via the `qualityLevels`
```

Comment:
Yeah, I was wondering about this myself since it's the representations API from the point of view of VHS, but the `qualityLevels` API from the point of view of the programmer. I will do as you suggest.

Reply:
correct, but ONLY if the quality levels plugin is included, which is optional. Manual rendition selection is still possible from the programmer point of view using the representations API. I suggested linking to the readme section since it mentions both the quality levels api (recommended approach) and the representations api

Reply:
Yep, covers all bases nicely.